src/sardana/macroserver/macros/standard.py

```diff
@@ -41,6 +41,8 @@
     ViewOption, iMacro, Hookable
 from sardana.macroserver.msexception import StopException
 from sardana.macroserver.scan.scandata import Record
+from sardana.macroserver.msparameter import Optional
```

Comment:
The idea is to imprort it from sardana.macroserver.macro as in the example.

Reply:
Provided in [43eabb4](https://github.com/sardana-org/sardana/pull/940/commits/43eabb45148e4ea0512a146f7b7cb7577bf0b237) commit.